packages/db/src/models/base.ts

```diff
@@ -17,6 +17,32 @@ class BasePgModel<ItemType, RecordType extends { cumulus_id: number }> {
     this.tableName = tableName;
   }
 
+  /**
+   * Fetches multiple items from Postgres
+   *
+   * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
+   * @param {Partial<RecordType>} params - An object or any portion of an object of type RecordType
+   * @param {updatedAtRange} updatedAtParams - An object with Date search bounds for updatedAt
+   * @returns {Promise<PostgresCollectionRecord[]>} List of returned records
+   */
+  async searchWithUpdatedAtRange(
```

Comment:
Don't we want to update the type constraint on the class to this now? So that we can ensure using this method on the base class is safe? It's probably only passing type safety now because nothing in the method explicitly uses `params.updated_at`

```
RecordType extends { cumulus_id: number, updated_at: Date }
```

Reply:
Right you are, I just missed it  -- nice catch.  692f544db

Reply:
Also : 2d35ea4a4, open to a different name on the type.